### ISSUE TITLE
Remove required strict equal

### ISSUE THREAD

**Issue by sirenkovladd** [putoutjs/minify#16]
- **Version** (`npm -v`): 10.5.2
- **Node Version** `node -v`: v20.13.1
- **OS** (`uname -a` on Linux): Darwin arm64

remove required strict equal

```js
import { minify } from "@putout/minify";

const wrap = (name, v) => {
  eval(`
    let o,t=0,c='';
    ${v}
    console.log("${name}:", o);
  `)
}

const source = `o=t===c?1:2`;
wrap('source', source)
const result = minify(source);
console.log(result);
wrap('result', result)
```

```sh
❯ node test.js
source: 2
o=t==c?1:2;
result: 1
```

**Comment by coderaiser** [comment]
[Added an option](https://github.com/putoutjs/minify/releases/tag/v3.16.0) 🎉, just use:

```js
import {minify} from "@putout/minify";

minify(code, {
    convertStrictEqualToEqual: false,
});
```

Is it works for you?

**Comment by sirenkovladd** [comment]
thank you

**Comment by sirenkovladd** [comment]
But I think that this option should be disabled by default, since using it will lead to errors and a bad experience

**Comment by coderaiser** [comment]
Submit PR https://github.com/putoutjs/minify/issues/21#issuecomment-2159960333, and I merge and publish major version.

**Comment by coderaiser** [comment]
Landed in [v4.0.0](https://github.com/putoutjs/minify/releases/tag/v4.0.0) 🎉
